public/landingpage.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Welcome</title>
+        <link rel="stylesheet" type="text/css" href="/assets/bootstrap.min.css"/>
+        <link rel="stylesheet" type="text/css" href="/assets/bootflat.min.css"/>
+        <link rel="stylesheet" type="text/css" href="/assets/startPage.css"/>
+        <style>
+            html, body {
+                height: 100%;
+                background-image: url(/assets/sunrise.jpg);
+                background-size: cover;
+                background-repeat: no-repeat;
+
+            }
+            #main-view {
+                position: relative;
+                top: 50%;
+                transform: translateY(-50%);
+                z-index: 1000;
+                max-width: 400px;
+            }
+
+            #main-view-container {
+                height: 100%;
+                width: 100%;
+                padding-left: 100px;
+                text-align: left;
+            }
+
+            h3 {
+                color: #ffffff;
+            }
+
+            p {
+                margin-bottom: 30px;
+                color: #efefef;
+            }
+
+            #tag {
+                text-align: left;
+            }
+
+            #background-overlay {
+                position: absolute;
+                background: #555;
+                width: 100%;
+                height: 100%;
+                z-index: 900;
+                opacity: 0.7;
+            }
+
+            #title {
+                margin-bottom: 30px;
+            }
+        </style>
+
+    </head>
+    <body>
+        <div id="background-overlay"></div>
+        <div id="main-view-container" class="container">
+            <div id="main-view" ui-view="content">
+                <h3 id="title">PenPal Gladiators</h3>
+                <p id="tag">
+                    Politics is a complicated topic. People have disagreements and it's hard to agree on something. But what if you could talk about it? To be able to discuss differences with someone of opposing opinion?
```

Comment:
We probably want to refine what we're saying here but it's good as a placeholder/for now